cli/main.go

```
@@ -82,6 +94,72 @@ func main() {
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return err
+		}
+		if hdr.Typeflag == tar.TypeDir {
+			continue
+		}
+		outName := strings.TrimPrefix(hdr.Name, "yamls/")
+		outPath := filepath.Join(dest, outName)
+		os.MkdirAll(filepath.Dir(outPath), 0700)
```

golangci-lint (1.64.8)

Error return value of `os.MkdirAll` is not checked

(errcheck)